composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "traderinteractive/filter-bool",
+    "description": "A collection of boolean filters.",
+    "keywords": [
+        "boolean",
+        "bool"
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "license": "MIT",
+    "require": {
+        "php": "^7.0",
+        "traderinteractive/exceptions": "^1.0"
+    },
+    "require-dev": {
+        "php-coveralls/php-coveralls": "^1.0",
+        "phpunit/phpunit": "^6.0",
+        "squizlabs/php_codesniffer": "^3.2"
+    },
+    "autoload": {
+        "psr-4": { "TraderInteractive\\": "src/" }
```

Comment:
If you change this to 
```
"psr-4": { "TraderInteractive\\Filter\\" : "src" }
```
you can put all of your classes in the `src/` diretory instead of `src/Filter/`